src/Process/Build/RefreshModules.php

```diff
@@ -43,12 +43,13 @@ public function __construct(
      */
     public function execute()
     {
-        $this->logger->info('Reconciling installed modules with shared config.');
+        $this->logger->notice('Reconciling installed modules with shared config.');
 
         try {
             $this->config->refresh();
         } catch (ShellException $exception) {
             throw new ProcessException($exception->getMessage(), $exception->getCode(), $exception);
         }
+        $this->logger->notice('End of reconciling modules.');
```

Comment:
Have you considered using the same text in the start and end text? It might be easier to follow, especially when doing full text searches. Your "Running DI compilation" process already follows this convention.

Example:
"Reconciling installed modules with shared config."
"End of reconciling installed modules with shared config."
"Validating configuration"
"End of validating configuration"

Reply:
That's actually what I did with every other message but for whatever reason, these two sets just felt clunky to me.  If you're thinking that sounds good, I'll make the change because I do like that convention.

Reply:
Fixed.

Reply:
I don't think it's too bad even in those cases. I think the benefits it provides to full text searches are well worth it.

Reply:
I'm fine either way, I just think it would be a nice addition.